config/postcreate-hooks/kubestellar.yaml

```diff
@@ -184,7 +184,7 @@ spec:
               - kubestellar
               - oci://ghcr.io/kubestellar/kubestellar/controller-manager-chart
               - --version
-              - "0.20.0-alpha.1"
+              - "v0.20.0-rc1"
```

Comment:
Shouldn't this be a placeholder?

Reply:
Oops, sorry, my original comment here was wrong. #1758 currently says this will be manually managed, not a placeholder.